Procfile.dev

```diff
@@ -1,4 +1,4 @@
 server: bin/rails server
-assets: bin/webpack-dev-server
+assets: WEBPACK_DEV_SERVER_PORT=3035 bin/webpack-dev-server
```

Comment:
At the moment the development-environment sets `WEBPACKER_DEV_SERVER_PUBLIC`. I guess we should change that to be `WEBPACKER_DEV_SERVER_PORT` instead?

https://github.com/exercism/development-environment/blob/99a722e84b37d298eb6451e3755b75e9a65e7524/stack.default.yml#L24

I think what you have here is correct, but I just want to check this all makes sense end-to-end before merging.

Reply:
Yeah. That sounds good. Is the `WEBPACKER_DEV_SERVER_PUBLIC` an actual env variable for webpacker configuration? Or is it something that we just created?

Reply:
Cool. Sounds good! I will open an issue to set `WEBPACK_DEV_SERVER_PORT` in the development environment.